### DISCUSSION TITLE
Project deletion

### DISCUSSION THREAD

**Discussion by gbarbon** [mia-platform/community#233] [Q&A] [answered]
### Which section?

Console

### Which is the version?

10.8

### Description

What should I do to delete a project on the Console? Not a production one, but, for instance, let's suppose I created an internal  project for demo/testing purposes that's not needed anymore and can be deleted. Should I open a SR, or there's a dedicated procedure to do this?

**Comment by davidebianchi** [comment, marked as answer]
Hi @gbarbon! At the moment, this is possible only if you (or someone else in your company) have access to the Console CMS. In the future we will expose an endpoint to delete a project and all the related information.
If no one have access to Console CMS, you can open a SR and ask to us to remove the project.

**Reply by gbarbon** [reply]
Ok thanks!
